src/lib/cache.js

```diff
@@ -48,19 +48,19 @@ class CacheClient {
     }
   }
 
-  get (key) {
+  async get (key) {
     if (enabled) {
-      return catboxMemoryClient.get(key)
+      return await catboxMemoryClient.get(key)
     }
     return null
   }
 
-  set (key, value) {
-    catboxMemoryClient.set(key, value, parseInt(ttl))
+  async set (key, value) {
+    await catboxMemoryClient.set(key, value, parseInt(ttl))
   }
 
-  drop (key) {
-    catboxMemoryClient.drop(key)
+  async drop (key) {
+    await catboxMemoryClient.drop(key)
   }
```

Comment:
Can't we return the promise directly here? We got recommendation from nodeJS expert regarding this and he suggested not to use await in this case. Return promise directly.